app/views/school_search/show.html.erb

```diff
@@ -55,7 +55,7 @@
           <%= f.govuk_collection_check_boxes(
               :partnership,
               [OpenStruct.new(
-                id: :partnered_with_another_provider, name: 'Show schools partnered with another provider')],
+                id: :in_a_partnership, name: 'Schools in a partnership')],
```

Comment:
"Show schools in a partnership"? 

Also, let's use double quotes on frontend strings (I know there were single quotes here, that's on me :/ )